src/DotNetty.Codecs/Base64/Base64Decoder.cs

```diff
@@ -21,7 +21,7 @@ public Base64Decoder(Base64Dialect dialect)
             this.dialect = dialect;
         }
 
-        protected override void Decode(IChannelHandlerContext context, IByteBuffer message, List<object> output) => output.Add(Base64.Decode(message, this.dialect));
+        protected internal override void Decode(IChannelHandlerContext context, IByteBuffer message, List<object> output) => output.Add(Base64.Decode(message, this.dialect));
```

Comment:
It is for https://github.com/StormHub/DotNetty/blob/UDP/src/DotNetty.Codecs/DatagramPacketEncoder.cs